book/content/open_research/open_research.md

```diff
@@ -4,7 +4,7 @@
 
 | Prerequisite | Importance | Notes |
 | -------------|----------|------|
-| Experience with version control | Helpful | Experience with GitHub is particularly useful |
+| [Experience with version control](/version_control/version_control.html) | Helpful | Experience with GitHub is particularly useful |
```

Comment:
Suggested change dropping the `.html` because we're _actually_ linking to an `.md` file so I think its more logical to have no suffix rather than the one that only exists at the end 😄 
```suggestion
| [Experience with version control](/version_control/version_control) | Helpful | Experience with GitHub is particularly useful |
```